docroot/sites/miller.lab.uiowa.edu/settings/default.local.settings.php

```diff
@@ -149,4 +149,4 @@
  */
 $settings['trusted_host_patterns'] = [
   '^.+$',
-];
\ No newline at end of file
+];
```

Comment:
??

Reply:
I think the provision process did that. not sure how it got that way in the first place.